src/test_application/activescenes/scenarios.cpp

```diff
@@ -191,27 +191,28 @@ static ScenarioMap_t make_scenarios()
         auto &rTags             = mainView.m_rTags;
         Builder_t builder{rTags, mainView.m_rTasks, mainView.m_rTaskData};
 
-        sceneOut.resize(21);
+        sceneOut.resize(24);
```

Comment:
i'd say move this constant 24 out to a static constexpr variable

Reply:
This number is related to the variables declared the very next line. In this case, moving the constant elsewhere hurts readability, though it seems best for me to just wrap the two lines into a template function.